var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -25,12 +24,15 @@ class PyMatplotlib(PythonPackage):
     ]
 
     license("Apache-2.0")
+    maintainers("adamjstewart")
```

Comment:
@rgommers would you be interested in maintaining this one too? It seems very similar to numpy/scipy now that it also uses a meson build system.

Reply:
Yes, I'd be happy to. I don't know as much about the Matplotlib, but there's a lot of commonality indeed so hopefully I can be helpful.